build_steps.py

```diff
@@ -411,6 +411,52 @@ def run_testament(platform):
         )
     ]
 
+@inject_paths
+def upload_release(platform):
+    test_url = "test-data/{buildername[0]}/{got_revision[0][nim]}/"
+    test_directory = 'public_html/' + test_url
+
+    nim_exe = 'build' / 'nim.exe'
+    nim_exe_dest = gen_dest_filename('nim.exe')
+
+    return [
+        ShellCommand(
+            name              = 'Bootstrap Release Version of Nim '
+            description       = 'Booting',
+            descriptionDone   = 'Booted',
+            descriptionSuffix = ' Release Nim',
+            command           = ['koch', 'boot', '-d:release'],
```

Comment:
Using `nim c -d:release compiler/nim.nim` only bootstraps the compiler once (instead of the two or more times that koch performs) and places the executable in the 'build/compiler' directory. `koch boot -d:release` will overwrite the executable in the 'build/bin' directory.